visuald/dpackage.d

```diff
@@ -1502,13 +1502,15 @@ class GlobalOptions
 			// $(WindowsSdkDir)\bin needed for rc.exe
 			// $(VCInstallDir)\bin needed to compile C + link.exe + DLLs
 			// $(VSINSTALLDIR)\Common7\IDE needed for some VS versions for cv2pdb
-			DMD.ExeSearchPath = r"$(VCInstallDir)\bin;$(VSINSTALLDIR)\Common7\IDE;$(WindowsSdkDir)\bin;$(DMDInstallDir)windows\bin";
-			DMD.ExeSearchPath64     = DMD.ExeSearchPath;
+            string windowsSdkX86Bin = r"$(WindowsSdkDir)bin\x86";
+            string windowsSdkX64Bin = r"$(WindowsSdkDir)bin\x64";

```

Comment:
These paths depend on the Windows SDK installed. They seem to have changed with the Windows 8 SDK. It's probably best to check for an existing `rc.exe` similar to `getDefaultLibPathCOFF32()`.